pillar/params.sls

```diff
@@ -75,3 +75,7 @@ proxy:
 reboot:
   group:          'default'
   directory:      'opensuse.org/rebootmgr/locks'
+
+transactional-update:
+  timer:
+    OnCalendar: '*:0/15' # See https://www.freedesktop.org/software/systemd/man/systemd.time.html#Calendar%20Events for syntax
```

Comment:
This is just a variable you are using in a template that is replaced inside a line that starts with `OnCalendar`. Otherwise, we would be using many different conventions depending on _where_ we are using those replacements...